src/App.js

```diff
@@ -32,6 +33,11 @@ export class App extends Component {
     isOver: PropTypes.bool.isRequired
   }
 
+  constructor (props) {
+    super(props)
+    props.doSetupLocalStorage()
```

Comment:
This does not seem to work on initial page load.
If you use empty browser profile and load webui, and then go to ipld explorer and click on wikipedia example you will see spinner forever, and no request:

>  ![2024-09-24_00-25_1](https://github.com/user-attachments/assets/b8922859-9297-4b6a-9b1a-789313e9a235)
>  ![2024-09-24_00-25](https://github.com/user-attachments/assets/9a8c96f8-0d01-4d89-8df7-e5c3aa7dc488)

If you then force-reload the page (F5) the helia will initialize with local gateway and work fine:

> ![2024-09-24_00-29](https://github.com/user-attachments/assets/ef46054c-c066-4702-9a73-6332d0a78106)

Reply:
I think landing directly on explore page might cause the issue? 

Issue doesn't happen if you land on status page first

Reply:
Might need to call the localStorage fn in router / component loader

Reply:
landing directly on explore page works for me with the latest.

https://github.com/user-attachments/assets/c7953852-56af-48e2-840b-3b41c76f83d3

Reply:
Yep, that was it. New version works fine.

ps. TIL [Firefox does not support HEVC while it is encumbered by patents](https://www.reddit.com/r/firefox/comments/zeg2qy/firefox_doesnt_support_h265_hevc_for_patent/), so inline player does not work (but i played fine with external player): 

> ![image](https://github.com/user-attachments/assets/d1d36de2-c521-45d5-9378-49e50f97719e)